File: frontend/recipeapp/lib/models/user_model.dart
class UserModel {
  String id;
  String userName;
  String firstName;
  String lastName;
  String membership;

  UserModel({
    required this.id,
    required this.userName,
    required this.firstName,
    required this.lastName,
    required this.membership,
  });
}

// ID              uuid.UUID `gorm:"type:char(36);primary_key;"`
// UserName        string    `gorm:"unique"`
// Name            string
// Password        string
// PasswordVersion uuid.UUID `gorm:"type:char(36)"`
// MembershipId    string
// LanguageId      string

File: backend/recipeapp/models/userModel.go
package models

import (
	"github.com/google/uuid"
	"gorm.io/gorm"
)

type User struct {
	gorm.Model
	ID              uuid.UUID `gorm:"type:char(36);primary_key;"`
	UserName        string    `gorm:"unique"`
	FirstName       string
	LastName        string
	Password        string
	PasswordVersion uuid.UUID `gorm:"type:char(36)"`
	MembershipId    string
	LanguageId      string
}
